docs/articles/nunit/writing-tests/constraints/GreaterThanConstraint.md

```diff
@@ -21,6 +21,7 @@ Is.Positive // Equivalent to Is.GreaterThan(0)
 ...Using(IComparer comparer)
 ...Using<T>(IComparer<T> comparer)
 ...Using<T>(Comparison<T> comparer)
+...Within(object tolerance)
```

Comment:
@OsirisTerje just curious before I merge this -- is this this technically `object` here? or should we provide a few different examples (e.g. TimeSpan, decimal, etc.)

Reply:
So the type is `object`, and gets interpreted at runtime. I took this example from the equality constraint.

Reply:
I can add examples of tolerance using numeric types, which was possible before the DateTime PR this documentation change is related to, but never documented here.